core/storage/file/gae_models.py

```diff
@@ -49,31 +58,96 @@ def get_new_id(cls, entity_name):
 
     @classmethod
     def get_undeleted(cls):
+        """Returns a list of undeleted FileMetadataModel instances. At most
+        feconf.DEFAULT_QUERY_LIMIT results are returned.
+
+        Returns:
+            list. Contains all undeleted instances of this class.
+        """
         return cls.get_all().filter(cls.deleted == False).fetch(  # pylint: disable=singleton-comparison
             feconf.DEFAULT_QUERY_LIMIT)
 
     @classmethod
     def _construct_id(cls, exploration_id, filepath):
+        """Constructs and returns an id string uniquely identifying the given
+        exploration and filepath.
+
+        Args:
+            exploration_id: str. The id of the exploration.
+            filepath: str. The path to the relevant file within the
+                exploration.
+
+        Returns:
+            str. Uniquely identifying string for the given exploration and
+                filepath (concatenation of exploration id and filepath).
```

Comment:
I think in general we don't need to indent for Returns, since there's always just one return value. (I don't really mind either way, but let's just make sure we're consistent with the rest of the codebase.